### ISSUE TITLE
RSS feeds are broken when using TYPO3 and Nginx

### ISSUE THREAD

**Issue by stucki** [drud/ddev#2403]
**Describe the bug**
When using TYPO3 and Nginx, any RSS feeds that should be served by TYPO3 will return a HTTP 404 error.

**To Reproduce**

1. Open any (non-existing) RSS URL on the ddev webserver.
2. Alternatively, set up a news feed including RSS feed in TYPO3

**Expected behavior**
It should return the output as configured by TYPO3. Instead, what I get is a HTTP 404 error by Nginx.

**Additional context**
The issue seems to be caused by the `location ~* \.(?:rss|atom)$ {` block in `nginx_typo3.conf`. Defining a new `location` means that the `location / {` block won't have effect here:

https://github.com/drud/ddev/blob/3976b9431b9b87101ba2416268f6f3cbfe317ffd/pkg/ddevapp/webserver_config_packr_assets/nginx-site-typo3.conf#L52-L56

The fix is to duplicate the contents from this block and define it again. (Alternatively, the code block which causes the issue could be removed, but then the `expires` setting is gone...)

**Comment by rfay** [comment]
Thanks so much for the report!

I assume this isn't new; I don't know of any changes that would have affected this.

You didn't fill out the required information about ddev version, etc. I assume this is ddev v1.15.0?

If you know the fix, you can easily demonstrate it by changing the .ddev/nginx_full/nginx-site.conf (don't forget to remove the #ddev-generated line at the top). Please do that and then submit a PR. 

But please provide full manual testing instructions.

Please give full recreation instructions

**Comment by gilbertsoft** [comment]
I see no reason for this definition, if you setup a rss feed with TYPO3 the expiration is defined anyway by TYPO3 and we will have the next issue... Also the next lines concerning the caching of images should be removed.
Imho we safely can remove the lines https://github.com/drud/ddev/blob/3976b9431b9b87101ba2416268f6f3cbfe317ffd/pkg/ddevapp/webserver_config_packr_assets/nginx-site-typo3.conf#L52-L63, do you agree @stucki ?

**Comment by stucki** [comment]
@gilbertsoft I agree that the feed rule can be removed. The other part about media rules won't hurt here, but it's also not needed on a local system. Anyway, it is not related to this problem.

@rfay Yes it's not new at all. I could reproduce this with ddev 1.15.3, but the configuration exists at least since 2018: https://github.com/drud/ddev/commit/e36b1f6eba25df2e761271d368282333db946306

Testing instructions: It's simple, because you don't even need to have TYPO3 running for it. Any request to https://typo3.ddev.site/test.rss should print the TYPO3 error about site not being configured. However, what you get is a 404 error produced by Nginx.

I will create a PR that removes these lines not only for TYPO3 but for all other configurations, too.
